src/components/security_manager/test/ssl_context_test.cc

```diff
@@ -378,20 +388,23 @@ TEST_F(SSLTest, OnTSL2Protocol_BrokenHandshake) {
 // TODO {AKozoriz} : Unexpected uncomplited init of SSL component.
 // In this and next tests.
 // Must be fixed after merge to develop.
-TEST_F(SSLTest, DISABLED_OnTSL2Protocol_Positive) {
+TEST_F(SSLTest, OnTSL2Protocol_Positive) {
   ASSERT_EQ(client_ctx->StartHandshake(&kClientBuf, &client_buf_len),
             security_manager::SSLContext::Handshake_Result_Success);
   ASSERT_FALSE(NULL == kClientBuf);
```

Comment:
@VVeremjova Maybe this check is unnecessary? Since you are checking this condition in the following while block:
   ASSERT_TRUE(NULL != kClientBuf);